src/aap_eda/settings/default.py

```diff
@@ -391,8 +393,15 @@ def _rq_common_parameters():
             "HOST": REDIS_HOST,
             "PORT": REDIS_PORT,
         }
-        if REDIS_CLIENT_CERT_PATH:
+        if REDIS_TLS:
             params["SSL"] = True
+        else:
+            # This check should be deprecated in favor of just using
```

Comment:
I'd like this marked as TODO.
```suggestion
            # TODO: Deprecate implicit setting based on cert path in favor of MQ_TLS as the determinant.
```